### PR TITLE
Update strings.txt

### PR DESCRIPTION
**Pull request by liberii**
I have completed the translation for Italian and Romanian.

Signed-off-by: liberii <eposta1@pm.me>

### CODE REVIEW

**Comment by pastk** [comment]
Thanks a lot!!!
Could you please sign your commit?
Add something like `Signed-off-by: Konstantin Pastbin <konstantin.pastbin@gmail.com>` to the commit message, see an example commit here: https://github.com/organicmaps/organicmaps/pull/1798/commits/f9ad1951b08a6445aa9be5623ba48fec9d570e4c

**Comment by pastk** [comment]
To the reviewer:
this change reverts `no` to `nb` lang change https://github.com/organicmaps/organicmaps/pull/1771/commits/57e2a77f9ffb9b274ec7bfce828bddf3d8b5bbd1
we might need to re-do it manually
otherwise the patch looks good to me format-wise

File: data/strings/strings.txt [hunk 1]
```diff
@@ -16282,15 +16314,15 @@
...
+    it = Attendere�
     ja = お待ちください…
     ko = 잠시만 기다려주십시오…
     nb = Vennligst vent…
     nl = Even geduld aub…
     pl = Proszę czekać…
     pt = Por favor aguarde�
     pt-BR = Espere, por favor�
-    ro = Te rog asteapta…
+    ro = Așteaptă�
```
Comment (biodranik) [inline data/strings/strings.txt]: What are these symbols at the end here and above?

File: data/strings/strings.txt [hunk 2]
```diff
@@ -22266,4 +22318,4 @@
     uk = Маршрут
     vi = Dấu chân
     zh-Hans = 追踪
-    zh-Hant = 追踪
+    zh-Hant = 追踪
```
Comment (biodranik) [inline data/strings/strings.txt]: ?

**Review by biodranik — APPROVED** [review]
Wow, a lot of work, thanks!

File: data/strings/strings.txt [hunk 3]
```diff
@@ -16896,15 +16934,15 @@
     fr = Paramètres de suivi
     hu = Követési beállítások
     id = Pengaturan treking
-    it = Impostazioni di tracciamento
+    it = Impostazioni percorso
     ja = 追跡の設定
     ko = 추적 설정
-    nb = Turinnstillinger
     nl = Tracking-instellingen
+    no = Turinnstillinger
```
Comment (biodranik) [inline data/strings/strings.txt]: Can you please revert no back to nb everywhere?

File: data/strings/strings.txt (hunk 2 above)
Comment (pastk) [reply data/strings/strings.txt]: FYI we can ignore the whitespace, it will be re-formatted by generate_localizations.sh (strings_utils.py) anyway

**Review by rtsisyk — APPROVED** [review]

**Comment by biodranik** [comment]
That one requires post-fixes. @pastk can you please help?

**Comment by pastk** [comment]
> That one requires post-fixes. @pastk can you please help?

Yeap, I'll do it.

**Comment by rsippl** [comment]
No doubt this commit involved a lot of work. Unfortunately, it introduced quite some errors in the Romanian spelling, by ignoring the spelling reform from 1993. From https://en.wikipedia.org/wiki/Romanian_alphabet: The sound /ɨ/ "is always spelled as â, except at the beginning and the end of words, where î is to be used instead."
For example, this commit replaces the correct form "Când" with the wrong one "Cînd", or "sunt" with "sînt". As this reform is nearly 30 years old, this is quite painful to the eyes of a native speaker. @liberii, can you please explain?

**Comment by biodranik** [comment]
Maybe there are two different language codes for both different spelling cases? like `ro-NEW`, or `ro-OLD`?

**Comment by biodranik** [comment]
> Even though this reform is almost 30 years old it is considered by experts to be a big mistake.

It would be great to check this in more detail. Do you have any links to read?

**Comment by rsippl** [comment]
> There is no law that imposes a certain type of writing.

@liberii, actually there is, 752/2001 states that Academia Română is responsible for the spelling rules, which are mandatory. This organization has published books (DOOM, ÎOOP etc.) which contain these rules. Of course there's no law which prohibits any way of spelling. As soon as you and the experts you mentioned can persuade Academia Română to change those rules, I'll happily acknowledge that your translation is correct.

**Comment by liberii** [comment]
> Maybe there are two different language codes for both different spelling cases? like `ro-NEW`, or `ro-OLD`?

Yes, there are two language codes: ro-RO and ro
Usually an application has both language codes.

> > Even though this reform is almost 30 years old it is considered by experts to be a big mistake.
> 
> It would be great to check this in more detail. Do you have any links to read?

https://dexonline.ro/articol/Alf_Lombard_-_Despre_folosirea_literelor_%C3%AE_%C8%99i_%C3%A2
http://www.philippide.ro/distorsionari_2008/219-227%20MANEA%20Constantin.pdf
https://web.archive.org/web/20210304144901/http://www.romlit.ro/index.pl/de_ce_scriu_i_susin_scrierea_cu__din_i
http://georgepruteanu.ro/103deceidini.htm
https://web.archive.org/web/20210419021649/https://arhiva.romanialiterara.com/index.pl/drag_academie
https://web.archive.org/web/20210304154527/http://www.romlit.ro/index.pl/schimbri_spectaculoase
https://web.archive.org/web/20210419040257/http://arhiva.romanialiterara.com/index.pl/ortografia_-_o_problem_de_istorie_a_culturii_romneti

**Comment by biodranik** [comment]
`ro` code is a more generic version of `ro-RO`. As far as I understand, `ro` can be used by Romanians who live abroad (`ro-US`, `ro-DE`, etc.), white ro-RO will be set by default for Romania's residents.
Will it work if we split "old" and "new" spelling into these two different categories of people? With some minor code changes in Organic Maps, it should be possible to have all basic translations in `ro`, and only define different words/sentences in `ro-RO`.

**Comment by liberii** [comment]
> `ro` code is a more generic version of `ro-RO`. As far as I understand, `ro` can be used by Romanians who live abroad (`ro-US`, `ro-DE`, etc.), white ro-RO will be set by default for Romania's residents. Will it work if we split "old" and "new" spelling into these two different categories of people? With some minor code changes in Organic Maps, it should be possible to have all basic translations in `ro`, and only define different words/sentences in `ro-RO`.

It would be nice to have two translations: ro and ro-RO; like pt and pt-BR, so that users can choose which language they want (in ro would be the translation with "î" and "sînt", and in ro-RO the translation with "â", "sunt" and other specific terms desired by @rsippl ). Right now, unfortunately, in Organicmaps for Android, it is not possible to choose the desired language. If it were possible to choose the language, it would make sense to introduce a new ro-RO translation. Would this be too complicated?

**Comment by biodranik** [comment]
iOS allows to choose the language and the region. Not sure about Android. It would be better to use a system setting instead of an app setting for the interface language, because the latter is complex to implement and may have some issues.

**Comment by liberii** [comment]
To change the language of the application will I need to change the system language? That works too. Where will the old spelling be? In ro or in ro-RO? Will the language of the voice instructions (sound.txt) be selectable between ro and ro-RO, because I don't like the current translation? If it will be possible to change the language of the app and the language of the voice instructions (sound.txt) I can make a new translation, but I need time. If it is not possible, then I give up.

**Comment by pastk** [comment]
> To change the language of the application will I need to change the system language? That works too. Where will the old spelling be? In ro or in ro-RO? Will the language of the voice instructions (sound.txt) be selectable between ro and ro-RO, because I don't like the current translation? If it will be possible to change the language of the app and the language of the voice instructions (sound.txt) I can make a new translation, but I need time. If it is not possible, then I give up.

I checked in Android 9 - two versions of Romanian are available: Romania and Republica Moldova, but its not what you're looking for I guess? Maybe there are more choices in higher Android versions.

Which spelling version (new or old) does the Android use itself?
It would make sense to align the app's version with the system.

**Comment by biodranik** [comment]
I thought that different spelling is pronounced in the same way. Was I wrong? Is there an "old" way to pronounce the same words and a "new" one?

**Comment by liberii** [comment]
>Which spelling version (new or old) does the Android use itself?

Android uses the new spelling version.

>I thought that different spelling is pronounced in the same way.

You're not wrong, it's pronounced the same way.
That's why it's considered a big mistake to put different letters for the same sound within the same word. This is not the main problem, but the translation itself. I wanted to make a new translation for "sound.txt" and "types_strings.txt". For "strings.txt" I did, but now @rsippl  is making some changes that I don't agree with (it would have been nice of him to consult with me, seeing that it is my work).

So, I really appreciate your willingness, but I understand from what we've discussed so far that there's no point in continuing here.
I wish you all the best and good luck with your work.

**Comment by owhyy** [comment]
> > `ro` code is a more generic version of `ro-RO`. As far as I understand, `ro` can be used by Romanians who live abroad (`ro-US`, `ro-DE`, etc.), white ro-RO will be set by default for Romania's residents. Will it work if we split "old" and "new" spelling into these two different categories of people? With some minor code changes in Organic Maps, it should be possible to have all basic translations in `ro`, and only define different words/sentences in `ro-RO`.
> 
> It would be nice to have two translations: ro and ro-RO; like pt and pt-BR, so that users can choose which language they want (in ro would be the translation with "î" and "sînt", and in ro-RO the translation with "â", "sunt" and other specific terms desired by @rsippl ). Right now, unfortunately, in Organicmaps for Android, it is not possible to choose the desired language. If it were possible to choose the language, it would make sense to introduce a new ro-RO translation. Would this be too complicated?

Hi. Although I get your point of view, I don't really agree with it. The fact that specialists and linguists don't agree with the spelling doesn't mean we have to add a whole new translation. At the end of the day, it's the Academia Română that specifies the spelling standard, and we must follow it, whether we like it or not. I found it quite weird to use „î” spelling in organicmaps, while every other app uses the „â” spelling, which is why I (not @rsippl) created the PR. If there's anything you don't agree with, you can add a comment to it, and we'll gladly discuss and modify it.

**Pull request merged**